charts/opentelemetry-target-allocator/templates/clusterRoleBinding.yaml

```
@@ -0,0 +1,20 @@
+{{- if .Values.targetAllocator.serviceAccount.create }}
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "helper.targetAllocatorClusterRoleBindingName" . }}
+  labels:
+    {{- include "helper.commonLabels" . | nindent 4 }}
+    app: {{ template "helper.name" . }}
+    chart: {{ template "helper.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

```suggestion
  labels:
    {{- include "helper.commonLabels" . | nindent 4 }}
```

I don't think we need the old style helm labels since we have equivalent labels in the commonLabels

app -> app.kubernetes.io/name
chart -> helm.sh/chart
release -> app.kubernetes.io/instance
heritage -> app.kubernetes.io/managed-by